docs/security.rst

```diff
@@ -104,3 +104,130 @@ vulnerabilities
 <https://github.com/pallets/flask/issues/248#issuecomment-59934857>`_, so
 this behavior was changed and :func:`~flask.jsonify` now supports serializing
 arrays.
+
+
+SSL/HTTPS
+---------
+
+For implementing HTTPS on your server
+
+Below some packages in suggestion order that implements this protocol:
```

Comment:
The phrasing `in suggestion order` isn't quite clear what you're trying to say... do you mean that you recommend the top ones more than the bottom ones? I'm hesitant to do that in our docs. IMHO, if a package isn't recommended, then we shouldn't even list it. 

Maybe just say `Below are some packages that implement this protocol`